crates/dc_bundle/src/definition/element.rs

```diff
@@ -462,13 +462,19 @@ impl Hash for FontStretch {
     }
 }
 
+impl NumOrVar {
+    pub const fn from_num(num: f32) -> Self {
+        NumOrVar { num_or_var_type: Some(NumOrVarType::Num(num)) }
```

Comment:
Unfortunately they need to be split. `from_var` needs things from figma_import so it can't be here, and `from_num` is used in dc_bundle so it needs to be here.

But I'll go through and fix the places where I missed using the function.

Reply:
I think there was only one other place that I'd missed.